### PR TITLE
37 travis ci integration

### LINKED ISSUE #37: travis CI integration

### PR DESCRIPTION
**Pull request by MarcoBgn**
closes #37

### CODE REVIEW

File: back-end/spec/features/player_song_name_spec.rb [hunk 1]
```diff
@@ -1,8 +1,17 @@
-feature 'Song name in player' do
-  scenario  'displays song name in player', js: true do
-    visit 'http://localhost:9000/'
-    click_link('Songs')
-    click_button('Play default song')
-    expect(page).to have_css("#song-name", text: "default song")
+feature	'displays song name' do
```
Comment (drjparry) [inline back-end/spec/features/player_song_name_spec.rb:1]: weird spacing...

File: back-end/spec/features/player_song_name_spec.rb [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+	scenario  'displays song name in player', js: true do
+		Artist.create(email: 'artyartison@gmail.com', password: '12345678')
+		visit 'http://localhost:9000/'
+		artist_log_in('artyartison@gmail.com', '12345678')
+		upload_track('test song')
+		sleep(3)
+		click_link('Home')
+		click_button('Play test song')
+		expect(page).to have_css("#song-name", text: 'test song')
+	end
```
Comment (drjparry) [inline back-end/spec/features/player_song_name_spec.rb:16]: indentation

File: front-end/app/views/main.html [hunk 3]
```diff
@@ -16,9 +16,10 @@
          </div>
        </div>
      </section>
+     
+      <div ng-repeat="song in player.songList | filter:query">
+        <div class="track_container"> <img ng-src={{song.image}}/> <span>{{song.id}} {{song.name}} <button id={{song.songId}} ng-click="player.changeSource(song.songId)">Play {{song.name}}</button> </span>
+        </div>
+     </div>
```
Comment (ghost) [inline front-end/app/views/main.html:24]: extra line

File: back-end/spec/upload_helper.rb [hunk 4]
```diff
@@ -12,6 +12,7 @@ def create_track_with_file_and_image(name="default")
 end
 
 def upload_track(name="default track")
+	click_link('Upload Sound')
 	fill_in "trackname", with: name
 	fill_in "genre", with: "Alternative"
 	attach_file 'file', Rails.root.join('spec','fixtures','testsong.mp3')
```
Comment (drjparry) [inline back-end/spec/upload_helper.rb:18]: indentation

File: front-end/app/styles/_main.scss [hunk 5]
```diff
@@ -64,7 +69,7 @@ body {
   text-align: center;
   border-bottom: 1px solid #e5e5e5;
   background-color: #f2fff5 !important;
-
+
```
Comment (ghost) [inline front-end/app/styles/_main.scss:72]: extra line?

File: front-end/app/scripts/controllers/main.js [hunk 6]
```diff
@@ -18,4 +18,7 @@ angular.module('fairMuseApp')
         }
       ];
     }
+  
+    self.user = localStorage.getItem('user_id')
+
```
Comment (drjparry) [inline front-end/app/scripts/controllers/main.js:23]: extra lines

File: back-end/spec/features/payments_spec.rb [hunk 7]
```diff
@@ -1,6 +1,7 @@
 feature 'Contribution payments' do
```
Comment (drjparry) [inline back-end/spec/features/payments_spec.rb:2]: extra lines

**Comment by drjparry** [comment]
:shipit:

**Pull request merged**